config.h

```diff
@@ -105,6 +105,7 @@ typedef struct {
     gboolean color_reversed; /** Color scheme, is reversed */
     gchar font_family[50]; /** Terminal font family */
     guint font_size;  /** Terminal font size */
+    gchar encoding[50]; /** Terminal encoding */
```

Comment:
If `iconv` is to be believed, the largest encoding in terms of character length is `CSISO60DANISHNORWEGIAN` with 22 characters:

```
iconv -l awk '{print length, $0}' | sort -nr | head -n 1
```
I think the two trailing slashes are for the trailing `\0`.